docs_zh_CN/deployment.md

```diff
@@ -21,44 +21,52 @@ python tools/deployment/pytorch2onnx.py
     --dynamic-export
 ```
 
-Description of arguments:
+所有参数的说明：
 
-- `model_config` : The path of a model config file.
-- `model_ckpt` : The path of a model checkpoint file.
-- `model_type` : The model type of the config file, options: `recog`, `det`.
-- `image_path` : The path to input image file.
-- `--output-file`: The path of output ONNX model. If not specified, it will be set to `tmp.onnx`.
-- `--device-id`: Which gpu to use. If not specified, it will be set to 0.
-- `--opset-version` : ONNX opset version, default to 11.
-- `--verify`: Determines whether to verify the correctness of an exported model. If not specified, it will be set to `False`.
-- `--verbose`: Determines whether to print the architecture of the exported model. If not specified, it will be set to `False`.
-- `--show`: Determines whether to visualize outputs of ONNXRuntime and pytorch. If not specified, it will be set to `False`.
-- `--dynamic-export`: Determines whether to export ONNX model with dynamic input and output shapes. If not specified, it will be set to `False`.
+| 参数               | 类型           | 描述                                                         |
+| ------------------ | -------------- | ------------------------------------------------------------ |
+| `model_config`     | str            | 模型配置文件的路径。                                         |
+| `model_ckpt`       | str            | 模型权重文件的路径。                                         |
+| `model_type`       | 'recog', 'det' | 配置文件对应的模型类型。                                     |
+| `image_path`       | str            | 输入图片的路径。                                             |
+| `--output-file`    | str            | 输出的 ONNX 模型路径。 默认为 `tmp.onnx`。                   |
+| `--device-id`      | int            | 使用哪块 GPU。默认为0。                                      |
+| `--opset-version`  | int            | ONNX 操作集版本。默认为11。                                  |
+| `--verify`         | bool           | 决定是否验证输出模型的正确性。默认为 `False`。               |
+| `--verbose`        | bool           | 决定是否打印导出模型的结构，默认为 `False`。                 |
+| `--show`           | bool           | 决定是否可视化 ONNXRuntime 和 PyTorch 的输出。默认为 `False`。 |
+| `--dynamic-export` | bool           | 决定是否导出有动态输入和输出尺寸的 ONNX 模型。默认为 `False`。 |
 
-**Note**: This tool is still experimental. Some customized operators are not supported for now. And we only support `detection` and `recognition` for now.
+:::{注}
```

Comment:
```suggestion
:::{note}
```
It is just a directive for our docs parser, so we don't need to translate it here.